src/pages/View/View.js

```diff
@@ -209,9 +209,12 @@ export default class View extends Component {
             //  - Add the dates together
             Promise.all(res[0].names.map((name, index) => {
                 
+                // the mpn65 palette is better (more distinct colours) but tol-rainbow has more colours
+                var palette_type = (res[0].names.length <= 65) ? ('mpn65') : ('tol-rainbow')
+
                 this.clinicians[name[0].label] = {
                     name: [name[0].label],
-                    color: colors[index],
+                    color: '#'.concat(palette(palette_type, res[0].names.length)[index])
```

Comment:
It's more efficient to create a palette object outside of the map function and index into the palette object inside the map function.